soh/soh/OTRGlobals.cpp

```diff
@@ -1943,9 +1943,14 @@ extern "C" CowIdentity Randomizer_IdentifyCow(s32 sceneNum, s32 posX, s32 posZ)
     return OTRGlobals::Instance->gRandomizer->IdentifyCow(sceneNum, posX, posZ);
 }
 
+extern "C" PotIdentity Randomizer_IdentifyPot(s32 sceneNum, s32 posX, s32 posZ) {
```

Comment:
It doesn't look like you are using this function in any .c file. I would recommend removing it and just using `Rando::Context::GetInstance()->` directly in `ShufflePots.cpp`

(The others for cow/beehive/fish etc are leftovers from pre-VB hookified days and can also be cleaned up in a new PR)